start-services.sh

```
@@ -0,0 +1,7 @@
+gnome-terminal -- bash -c "cd amrit-local-setup;sudo docker-compose up"
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)